Dockerfile.e2e

```diff
@@ -22,8 +22,8 @@ RUN contrib/download-frozen-image-v2.sh /output/docker-frozen-images \
 	hello-world:latest@sha256:be0cd392e45be79ffeffa6b05338b98ebb16c87b255f48e297ec7f98e123905c
 
 # Download Docker CLI binary
-COPY hack/dockerfile hack/dockerfile
-RUN hack/dockerfile/install-binaries.sh dockercli
+COPY hack/dockerfile/ hack/dockerfile
```

Comment:
Do you recall why you changed this? Should this be `/.` (slash-dot)? https://docs.docker.com/engine/reference/commandline/cp/#extended-description

Reply:
Not intentional.

Reply:
Fixed